src/handlers/check_commits/force_push_range_diff.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Context as _;
+
+use crate::config::RangeDiffConfig;
+use crate::github::GithubCompare;
+use crate::github::IssueRepository;
+use crate::github::IssuesEvent;
+use crate::handlers::Context;
+
+pub(super) async fn handle_event(
+    ctx: &Context,
+    host: &str,
+    _config: &RangeDiffConfig,
+    event: &IssuesEvent,
+    compare_after: &GithubCompare,
+) -> anyhow::Result<()> {
+    if !matches!(event.action, crate::github::IssuesAction::Synchronize) {
+        return Ok(());
+    }
+
+    let (Some(before), Some(after)) = (event.before.as_ref(), event.after.as_ref()) else {
+        tracing::warn!("synchronize event but no before or after field");
+        return Ok(());
+    };
+
+    let org = event.repository.owner();
```

Comment:
We can check if `forced` is `true` (https://docs.github.com/en/webhooks/webhook-events-and-payloads#issues) to avoid having to call `compare` for non-force pushes.

Reply:
Ah, I checked the wrong webhook, nevermind.